Animations/Zoom/Down.js

```diff
@@ -26,5 +28,5 @@ var ZoomDown = {
     'transform': 'scale(.1) translateY(-20px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('ZoomDown', ZoomDown);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('ZoomDown', ZoomDown);
```

Comment:
This does not need to have `'ZoomDown'` passed into it since we have defined a `className` property in `ZoomDown`.